### PR TITLE
Make authentication parameters configurable on the pulsar-client-kafka

### PR DESCRIPTION
**Pull request by ssunorz**
### Motivation

Make authentication parameters configurable on the pulsar-client-kafka.

### Modifications

Add authentication parameters to the PulsarClientKafkaConfig.

### Result

Users can set authentication parameters on the pulsar-client-kafka.

### CODE REVIEW

File: pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/pulsar/client/kafka/compat/PulsarClientKafkaConfig.java [hunk 1]
```diff
@@ -29,6 +30,8 @@
 
     /// Config variables
     public static final String AUTHENTICATION_CLASS = "pulsar.authentication.class";
+    public static final String AUTHENTICATION_PARAMS_MAP = "pulsar.authentication.params.map";
+    public static final String AUTHENTICATION_PARAMS_STRING = "pulsar.authentication.params.string";
```
Comment (merlimat) [inline pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/pulsar/client/kafka/compat/PulsarClientKafkaConfig.java]: @ssunorz Can you also add these new keys to the docs page at https://github.com/apache/incubator-pulsar/blob/master/site2/docs/adaptors-kafka.md ?

Comment (ssunorz) [reply pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/pulsar/client/kafka/compat/PulsarClientKafkaConfig.java]: @merlimat Thanks for the review. I updated the document according to your suggestions.

**Comment by nkurihar** [comment]
run java8 tests

**Comment by nkurihar** [comment]
run cpp tests

**Comment by merlimat** [comment]
run java8 tests

**Comment by srkukarni** [comment]
run java8 tests

**Comment by nkurihar** [comment]
run java8 tests

**Comment by nkurihar** [comment]
run java8 tests

**Comment by nkurihar** [comment]
run java8 tests

**Comment by nkurihar** [comment]
run java8 tests

**Pull request merged**